docs/src/pages/components/timeline/SimpleTimeline.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Timeline from '@material-ui/lab/Timeline';
+import TimelineItem from '@material-ui/lab/TimelineItem';
+import TimelineItemTail from '@material-ui/lab/TimelineItemTail';
+import TimelineItemContent from '@material-ui/lab/TimelineItemContent';
+import TimelineItemDot from '@material-ui/lab/TimelineItemDot';
+
+export default function SimpleTimeline() {
+  return (
+    <Timeline>
+      <TimelineItem>
+        <TimelineItemTail />
+        <TimelineItemDot />
+        <TimelineItemContent>Eat</TimelineItemContent>
+      </TimelineItem>
+      <TimelineItem>
+        <TimelineItemTail />
+        <TimelineItemDot />
+        <TimelineItemContent>Code</TimelineItemContent>
+      </TimelineItem>
+      <TimelineItem>
+        <TimelineItemTail />
+        <TimelineItemDot />
+        <TimelineItemContent>Sleep</TimelineItemContent>
+      </TimelineItem>
+      <TimelineItem>
+        <TimelineItemDot />
+        <TimelineItemContent>Repeat</TimelineItemContent>
+      </TimelineItem>
+    </Timeline>
```

Comment:
For the first demo, I think that we could display 3 instead of 4. It would shrink the demo enough to enable inline previews:

```suggestion
    <Timeline>
      <TimelineItem>
        <TimelineItemTail />
        <TimelineItemDot />
        <TimelineItemContent>Eat</TimelineItemContent>
      </TimelineItem>
      <TimelineItem>
        <TimelineItemTail />
        <TimelineItemDot />
        <TimelineItemContent>Code</TimelineItemContent>
      </TimelineItem>
      <TimelineItem>
        <TimelineItemDot />
        <TimelineItemContent>Sleep</TimelineItemContent>
      </TimelineItem>
    </Timeline>
```

<img width="863" alt="inline-preview" src="https://user-images.githubusercontent.com/3165635/83942938-69624f80-a7f8-11ea-9f21-03b1d565528b.png">